internal/authz/authz_test.go

```diff
@@ -33,40 +36,32 @@ func TestPartialStringArgs(t *testing.T) {
 		"bob",
 		"viewer",
 	}
-	if !reflect.DeepEqual(expArgs, args) {
-		t.Fatalf("unexpected args\n\ngot: %v\n\nexp: %v", args, expArgs)
+	if diff := cmp.Diff(expArgs, args, cmpopts.SortSlices(func(a, b any) bool { return a.(string) < b.(string) })); diff != "" {
```

Comment:
Ordering of that slice is irrelevant for the filter. It seems from some indeterminate ordering in the PE result's queries, I didn't chase that down, because it had no semantic implications there, either.